README.md

```diff
@@ -13,17 +13,64 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123');
+const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123', '&bar=456', '#heading-1');
 
-console.log(fullUrl);
+console.log(fullUrl.toString()); // 'http://www.google.com/a/b/cd?foo=123&bar=456#heading-1'
```

Comment:
@jonkoops Just noticed that there's an unnecessary `.toString()` called on `fullUrl` here. The return type of `urlJoin()` here is already `string`, right?

Reply:
It's still a `URL` instance from what I can seem, so I think a `toString()` here is valid.